src/main/java/tz/go/moh/him/thscp/mediator/epicor/orchestrator/EmergencySupplyChainCommoditiesStockStatusOrchestrator.java

```diff
@@ -0,0 +1,166 @@
+package tz.go.moh.him.thscp.mediator.epicor.orchestrator;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import org.codehaus.plexus.util.StringUtils;
+import org.openhim.mediator.engine.MediatorConfig;
+import org.openhim.mediator.engine.messages.MediatorHTTPRequest;
+import org.openhim.mediator.engine.messages.MediatorHTTPResponse;
+import tz.go.moh.him.mediator.core.domain.ErrorMessage;
+import tz.go.moh.him.mediator.core.domain.ResultDetail;
+import tz.go.moh.him.mediator.core.validator.DateValidatorUtils;
+import tz.go.moh.him.thscp.mediator.epicor.domain.EmergencySupplyChainCommoditiesStockStatusRequest;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class EmergencySupplyChainCommoditiesStockStatusOrchestrator extends BaseOrchestrator {
+
+    /**
+     * Initializes a new instance of the {@link DosProductOrchestrator} class.
```

Comment:
The link should be refactored to EmergencySupplyChainCommoditiesStockStatusOrchestrator